### DISCUSSION TITLE
AI解決曩昔收藏家的困擾

### DISCUSSION THREAD

**Discussion by ccathyy** [1101-GS3073-C/Python-Community-GS3073-C#93] [General]
AI的應用範圍極廣，倘若先前就有這項技術，那些收藏家就不會害怕買到贗品。
不知道現在AI技術可不可以幫我驗鞋子或是其他有品牌的商品的真假，
超愛網購的我很怕買到山寨品qq

文章:
https://www.techbang.com/posts/91161-ai-art-forgery

**Comment by wh20502** [comment]
我覺得如果是ai影像處理的話，標準化之類的影像處理應該是必備的，不然會受到環境光影響，造成判讀上的誤差。

**Comment by minniewei** [comment]
我覺得這項技術十分實用，在未來的各個領域都能發揚光大

**Comment by ghost** [comment]
讓我想到 究竟是AI辨識的進步幅度比較快 還是人類仿造贗品的進步速度比較快呢?XD

**Comment by Shiba010** [comment]
以後連鑑定專家也要被AI取代了嗎？！
